code/modules/uplink/uplink_items/uplink_badass.dm

```diff
@@ -24,32 +41,18 @@
 			Radio headset does not include encryption key. No gun included."
 	item = /obj/item/storage/box/syndie_kit/centcom_costume
 
-/datum/uplink_item/badass/claymore
-	name = "Claymore"
-	cost = 8
-	player_minimum = 25
-	desc = "A claymore. We don't know why you'd do this."
-	item = /obj/item/claymore
-
 /datum/uplink_item/badass/costumes/clown
 	name = "Clown Costume"
 	desc = "Nothing is more terrifying than clowns with fully automatic weaponry."
 	item = /obj/item/storage/backpack/duffelbag/clown/syndie
 
-/datum/uplink_item/badass/plastitanium_toolbox
-	name = "Plastitanium Toolbox"
-	desc = "A very high impact toolbox. Excels at destroying stationary structures."
-	item = /obj/item/storage/toolbox/plastitanium
-	cost = 2		//18 damage on mobs, 50 on objects, 4.5 stam/hit
-
 /datum/uplink_item/badass/balloon
 	name = "Syndicate Balloon"
 	desc = "For showing that you are THE BOSS: A useless red balloon with the Syndicate logo on it. \
 			Can blow the deepest of covers."
 	item = /obj/item/toy/syndicateballoon
 	cost = 20
 	cant_discount = TRUE
-	illegal_tech = FALSE
```

Comment:
You do know these were flagged as false because they're a feature on some stations right